### PR TITLE
DOC: Add missing examples to ``np.ma``

### PR DESCRIPTION
**Pull request by linus-md**
Relates to #21351. See also: #25348.

### CODE REVIEW

File: numpy/ma/core.py [hunk 1]
```diff
@@ -5541,6 +5541,17 @@ def round(self, decimals=0, out=None):
         --------
         numpy.ndarray.round : corresponding function for ndarrays
         numpy.around : equivalent function
+
+        Examples
+        --------
+        >>> import numpy.ma as ma
+        >>> x = np.ma.array([1.35, 2.5, 1.5, 1.75, 2.25, 2.75],
```
Comment (mtsokol) [inline numpy/ma/core.py:5548]: You imported `ma` so you can do `x = ma.array(....` for consistency with existing examples in `ma/core.py`.

File: numpy/ma/extras.py [hunk 2]
```diff
@@ -839,6 +839,16 @@ def compress_nd(x, axis=None):
     -------
     compress_array : ndarray
         The compressed array.
+
+    Examples
+    --------
+    >>> import numpy.ma as ma
+    >>> x = np.ma.array(np.arange(9).reshape(3, 3), mask=[[1, 0, 0],
```
Comment (mtsokol) [inline numpy/ma/extras.py:846]: Same as above.

File: numpy/ma/extras.py [hunk 3]
```diff
@@ -1268,6 +1307,16 @@ def setxor1d(ar1, ar2, assume_unique=False):
     --------
     numpy.setxor1d : Equivalent function for ndarrays.
 
+    Examples
+    --------
+    >>> import numpy.ma as ma
+    >>> ar1 = np.ma.array([1, 2, 3, 2, 4])
+    >>> ar2 = np.ma.array([2, 3, 5, 7, 5])
+    >>> np.ma.setxor1d(ar1, ar2)
```
Comment (mtsokol) [inline numpy/ma/extras.py:1315]: Same here, no need for `np.`.

File: numpy/ma/extras.py [hunk 4]
```diff
@@ -1303,6 +1352,14 @@ def in1d(ar1, ar2, assume_unique=False, invert=False):
     -----
     .. versionadded:: 1.4.0
 
+    Examples
+    --------
+    >>> import numpy.ma as ma
+    >>> ar1 = np.ma.array([0, 1, 2, 5, 0])
```
Comment (mtsokol) [inline numpy/ma/extras.py:1358]: Same here.

File: numpy/ma/extras.py [hunk 5]
```diff
@@ -1303,6 +1352,14 @@ def in1d(ar1, ar2, assume_unique=False, invert=False):
...
+    >>> ar2 = [0, 2]
+    >>> np.in1d(ar1, ar2)
```
Comment (mtsokol) [inline numpy/ma/extras.py:1360]: I think you should use `ma.in1d`. This example doesn't use `in1d` from the `ma` module.

File: numpy/ma/extras.py [hunk 6]
```diff
@@ -1344,6 +1401,16 @@ def isin(element, test_elements, assume_unique=False, invert=False):
     -----
     .. versionadded:: 1.13.0
 
+    Examples
+    --------
+    >>> import numpy.ma as ma
+    >>> element = np.ma.array([1, 2, 3, 4, 5, 6])
+    >>> test_elements = [0, 2]
+    >>> np.ma.isin(element, test_elements)
```
Comment (mtsokol) [inline numpy/ma/extras.py:1409]: Same here, no need for `np.`.

File: numpy/ma/extras.py [hunk 7]
```diff
@@ -1360,6 +1427,16 @@ def union1d(ar1, ar2):
     --------
     numpy.union1d : Equivalent function for ndarrays.
 
+    Examples
+    --------
+    >>> import numpy.ma as ma
+    >>> ar1 = np.ma.array([1, 2, 3, 4])
+    >>> ar2 = np.ma.array([3, 4, 5, 6])
+    >>> np.ma.union1d(ar1, ar2)
```
Comment (mtsokol) [inline numpy/ma/extras.py:1435]: Same here.

**Review by mtsokol — CHANGES_REQUESTED** [review]
Just a couple of nitpicks and it's ready to go!

File: numpy/ma/extras.py (hunk 7 above)
Comment (linus-md) [reply numpy/ma/extras.py]: All done @mtsokol!

**Review by mtsokol — APPROVED** [review]

**Comment by mattip** [comment]
Thanks @linus-md

**Pull request merged**